index.html

```diff
@@ -1319,6 +1333,49 @@ <h3><dfn>close()</dfn> method</h3>
       <li>Return |promise|.
     </ol>
   </section>
+  <section>
+    <h3><dfn>forget()</dfn> method</h3>
+
+    <aside class="example">
+      It is posssible to voluntarily revoke a permission to a serial port that was granted by a user.
+
+      <pre class="js">
+        // Request a serial port.
+        const port = await navigator.serial.requestPort();
+
+        // Then later... revoke permission to the serial port.
+        await port.forget();
+      </pre>
+    </aside>
+
+    The {{SerialPort/forget()}} method steps are:
+
+    <ol>
+      <li>
+        If the user agent can't perform this action (e.g. permission was granted
+        by administrator policy), return [=a promise resolved with=] `undefined`.
+      <li>Run the following steps [=in parallel=]:
+        <ol>
+          <li>Set [=this=].{{SerialPort/[[state]]}} to `"forgetting"`.
+          <li>
+            If [=this=].{{SerialPort/[[readable]]}} is not `null`, invoke
+            [=ReadableStream/cancel=] on [=this=].{{SerialPort/[[readable]]}}.
+          <li>
+            If [=this=].{{SerialPort/[[writable]]}} is not `null`, invoke
+            [=WritableStream/abort=] on [=this=].{{SerialPort/[[writable]]}}.
```

Comment:
We can remove these steps now that the stream algorithms know how to react to loss of permission.

Reply:
Done.